build/index.js

```diff
@@ -175,18 +177,29 @@ var Front = (function () {
         var path = url.replace(URL, '').replace(/^\//, '');
         return this.httpCall({ method: 'GET', path: path }, null, callback);
     };
-    Front.prototype.httpCall = function (details, params, callback) {
+    Front.prototype.httpCall = function (details, params, callback, retries) {
+        var _this = this;
+        if (retries === void 0) { retries = 0; }
```

Comment:
Is there a reason this is not `if (!retries)`? Even if it is 0 you will just reassign it to 0 so it won't make any difference, but it will catch other possible null-like values.

Reply:
This is exactly the semantic of TS for default values of args & I think that someone had previously forgot to commit these build changes in one of the previous PRs.